modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -6,7 +6,9 @@ endif::[]
 Provisioning a VPC for your organization consists of two major steps:
 
 . Create the VPC. +
-This is a self-service process. You can create a VPC using either the Runtime Manager UI, or the Anypoint Platform CLI.
+This is a self-service process. You can create a VPC using either the Runtime Manager UI, or the Anypoint Platform CLI. +
```

Comment:
I don't know the current state, but is this true today? I thought Support needs to enable self-service first

Reply:
NVM I just realized this is about VPCs, not VPNs